client/packages/inventory/src/Stocktake/DetailView/ContentArea/columns.tsx

```diff
@@ -173,6 +173,9 @@ export const useStocktakeColumns = ({
             { path: ['lines', 'packSize'] },
             { path: ['packSize'] },
           ]),
+        cellProps: {
+          defaultValue: UNDEFINED_STRING_VALUE,
+        },
```

Comment:
Changed this one, per the pattern we have elsewhere, to show `-` when the line is just a placeholder, value doesn't really exist :)